app/models/reports/percentage.rb

```diff
@@ -8,7 +9,9 @@ def percentage(numerator, denominator, with_rounding: true)
     end
   end
 
-  def rounded_percentages(counts_hash)
+  memoize def rounded_percentages(counts_hash)
```

Comment:
Since this is now memoized, should `appts_scheduled_rates` not be memoized any more?

Reply:
That's right